includes/acl/core/compressed_clip.h

```diff
@@ -84,4 +86,75 @@ namespace acl
 
 	static_assert(alignof(CompressedClip) == 16, "Invalid alignment for CompressedClip");
 	static_assert(sizeof(CompressedClip) == 16, "Invalid size for CompressedClip");
+
+	struct SegmentHeader
+	{
+		uint32_t				num_samples;
```

Comment:
formatting here appears messed up a bit, can you align the members/comments please?

Reply:
It's because GitHub is using eight spaces per tab, rather than four like Visual Studio does.

https://stackoverflow.com/a/33831598

I'll include an .editorconfig in my PR revisions.